src/DetailsView/components/assessment-issues-test-view.tsx

```diff
@@ -35,6 +35,7 @@ export type AssessmentIssuesTestViewDeps = {
 export type AssessmentIssuesTestViewProps = {
     deps: AssessmentIssuesTestViewDeps;
     assessmentStoreData: AssessmentStoreData;
+    includeStepsText?: boolean;
```

Comment:
Where is this used? I couldn't find any references.

Reply:
This property is referenced in the file default-test-view-container-provider.tsx , in the method createAssessmentAutomatedChecksTestViewContainer.

Reply:
Right, but then where else is it referenced? Is this property actually used by `AssessmentIssuesTestView` or is it a vestige of something old? I went through file history and see now that it is passed into `DetailsListIssuesView` and used there.

Reply:
Sorry I think my comment was confusing. I ended up finding that it *is* used when `...props` are passed into `DetailsListIssuesView` inside `AssessmentIssuesTestView` so we do need it.